### PR TITLE
Cleanup user-guide pages mu-t

### PR DESCRIPTION
**Pull request by kprosise**
The last pages of the User Guide requiring cleanup have been addressed. Simplified language and rst, style fixes.

QA Steps: Did not step through all instructions. Rendered HTML was inspected visually. Ran linter and linkcheck.

This commit applies to FFTK 988

## Readiness

*   [x] Merge (pending reviews)
*   [ ] Merge after _date or event_
*   [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._  
_You can fill this out after opening the PR. "Did I..."_

*   [x] Run spelling and grammar check, preferably with linter.
*   [x] Avoid changing any header associated with a link/reference.
*   [ ] Step through instructions (or ask someone to do so).
*   [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
*   [x] Match tone and style of page/section.
*   [x] Run `make linkcheck`.
*   [x] View HTML in a browser to check rendering.
*   [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
*   [x] follow best practices for commits.
    *   [x] Descriptive title written in the imperative.
    *   [x] Include brief overview of QA steps taken.
    *   [x] Mention any related issues numbers.
    *   [x] End message with sign off/DCO line (`-s, --signoff`).
    *   [x] Sign commit with your gpg key (`-S, --gpg-sign`).
    *   [x] Squash commits if needed.
*   [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._  
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 2bfbb0843d0664b0503400ed3681b7f0d6bc5da7 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2322/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for ce0fa3be5ff0281a7587cce9709b6fbfb548f24c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2323/docs/artifacts/html/index.html

File: source/user-guide/multi-stage-container/multi-stage-container.rst [hunk 1]
```diff
@@ -123,20 +100,18 @@ Create the ``Dockerfile`` that implements the single stage container:
     
     ENTRYPOINT ["/app/start.sh"]
 
-The ``Dockerfile`` is very simple. It installs ``build-essential``, 
-copies the files ``helloworld.c`` and ``start.sh`` to the container image, 
-compiles ``helloworld.c`` and sets the entrypoint to start the ``start.sh`` script.
+The ``Dockerfile`` is straightforward..
```
Comment (angolini) [inline source/user-guide/multi-stage-container/multi-stage-container.rst:103]:
```suggestion
The ``Dockerfile`` is straightforward.
```

File: source/user-guide/multi-stage-container/multi-stage-container.rst [hunk 2]
```diff
@@ -123,20 +100,18 @@ Create the ``Dockerfile`` that implements the single stage container:
...
+It installs ``build-essential``, copies the files ``helloworld.c`` and ``start.sh`` to the container image,
+then compiles ``helloworld.c`` and sets the entrypoint to start the ``start.sh`` script.
 
-Build the docker example and check the image size:
+Build the Docker example and check the image size:
```
Comment (angolini) [inline source/user-guide/multi-stage-container/multi-stage-container.rst:107]: I think here should be "Build the Docker image" or "Build the Docker sample image" as we are not building an example.

File: source/user-guide/multi-stage-container/multi-stage-container.rst [hunk 3]
```diff
@@ -226,70 +191,60 @@ Create the ``Dockerfile`` that implements the multi-stage container:
...
+Next, it installs ``build-essential`` and compiles ``helloworld.c``.
 
-The second stage starts with ``AS final-stage`` right after specifying the image used 
-for the second stage (line 18 of the Dockerfile).
-Finally, it ``COPY`` the ``helloworld`` binary from the first stage using the parameter ``--from=builder``.
+The second stage starts with ``AS final-stage`` after specifying the image to be used. (line 18 of the Dockerfile).
+Finally, ``COPY`` get the ``helloworld`` binary from the first stage using the parameter ``--from=builder``.
 
-Build the docker example and check the image size:
+Build the Docker example and check the image size:
```
Comment (angolini) [inline source/user-guide/multi-stage-container/multi-stage-container.rst:201]: the same

File: source/user-guide/multi-stage-container/multi-stage-container.rst [hunk 4]
```diff
@@ -226,70 +191,60 @@ Create the ``Dockerfile`` that implements the multi-stage container:
...
+::
 
      hello, world!
      hello, world!
      hello, world!
 
-In the second terminal, inspect the image and note that only the needed files are 
-present in the image (``helloworld.c`` is not installed in the final stage):
+In the second terminal, inspect the image.
+Note that only the needed files are present in the image (``helloworld.c`` is not installed in the final stage):
```
Comment (angolini) [inline source/user-guide/multi-stage-container/multi-stage-container.rst:230]:
```suggestion
Note that only the required files are present in the image (``helloworld.c`` is not installed in the final stage):
```

File: source/user-guide/submodule/submodule.rst [hunk 5]
```diff
@@ -1,37 +1,27 @@
...
-Working with Git Submodules
+Working With Git Submodules
 ===========================
 
-This section shows how to add a git submodule to your FoundriesFactory repository.
+This section shows how to add a git submodule to your FoundriesFactory® repository.
 
-This is very helpful when you want to use an external private or public git repository, 
-such as GitHub_, connected to the FoundriesFactory repository as a submodule.
+This is for when you want an external git repository, such as GitHub, connected to a Factory repository as a submodule.
```
Comment (angolini) [inline source/user-guide/submodule/submodule.rst:8]:
```suggestion
This is when you want an external Git repository, such as GitHub, connected to a Factory repository as a submodule.
```

**Review by angolini — APPROVED** [review]

File: source/user-guide/multi-stage-container/multi-stage-container.rst (hunk 2 above)
Comment (kprosise) [inline source/user-guide/multi-stage-container/multi-stage-container.rst:107]:
```suggestion
Build the Docker image and check the image size:
```

File: source/user-guide/multi-stage-container/multi-stage-container.rst (hunk 3 above)
Comment (kprosise) [inline source/user-guide/multi-stage-container/multi-stage-container.rst:201]:
```suggestion
Build the Docker image and check the image size:
```

**Pull request merged**